### PR TITLE
Update README.md for final GSoC 17 submission

### PR DESCRIPTION
**Pull request by connienguyen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +19,28 @@ If you are an authorized developer on this project, request access fo the `Googl
 If you are not yet an authorized developer on this project, you are still able to contribute, but will have to use your own `GoogleService-Info.plist` and `SecretKeys.plist` files. You can read more about the structure of the plist files on the [project wiki](https://github.com/systers/volunteers-iOS/wiki/Required-files).
 
 You may get an error when trying to log into Facebook if you are not on the list of the app's approved testers. To become a test user or developer user, request to become one and provide your facebook ID or username.
+
+## How to Contribute
+
+If you're interested in contributing to the project, follow these steps:
+
+For first time contributors, it's recommended to work on issues marked `Priority: LOW` or [unit tests](https://github.com/systers/volunteers-iOS/tree/development/VOLA/VOLATests).
```
Comment (bphenriques) [inline README.md]: or unit tests? Can't figure out what you mean

File: README.md [hunk 2]
```diff
@@ -14,3 +19,28 @@ If you are an authorized developer on this project, request access fo the `Googl
...
+
+1. Create a fork of the repo.
+2. Browse open [issues](https://github.com/systers/volunteers-iOS/issues) and select one you want to attempt.
+3. Make your changes for the issue on your fork of the repo as a new branch.
+4. Make a pull request to the Systers [development branch](https://github.com/systers/volunteers-iOS/tree/development) for the project. Don't forget to tag the issue you're fixing in your pull request and the [GSoC mentors and student](#maintainers-and-contributors) as reviewers for your pull request.
+
```
Comment (bphenriques) [inline README.md]: Yeah. This GSOC lacked commit conventions (my fault)

Ideally the maintainers should respect the following commit message.

```sh
fix #<number of commit>: <brief summary>

* Bulletpoint 1 
* Bulletpoint 2
```

I don't mean you to add this info. Just for us to keep in mind :)

File: README.md [hunk 3]
```diff
@@ -14,3 +19,28 @@ If you are an authorized developer on this project, request access fo the `Googl
...
+## How to Contribute
+
+If you're interested in contributing to the project, follow these steps:
+
+For first time contributors, it's recommended to work on issues marked `Priority: LOW` or [unit tests](https://github.com/systers/volunteers-iOS/tree/development/VOLA/VOLATests).
+
+1. Create a fork of the repo.
+2. Browse open [issues](https://github.com/systers/volunteers-iOS/issues) and select one you want to attempt.
+3. Make your changes for the issue on your fork of the repo as a new branch.
+4. Make a pull request to the Systers [development branch](https://github.com/systers/volunteers-iOS/tree/development) for the project. Don't forget to tag the issue you're fixing in your pull request and the [GSoC mentors and student](#maintainers-and-contributors) as reviewers for your pull request.
```
Comment (bphenriques) [inline README.md]: I would add anywhere to make sure that the unit tests are provided and that they are passing on a clean state. Moreover, there should no lint warning except when applicable (TODO, FIXME which IMO still need to be avoided)

File: README.md (hunk 2 above)
Comment (connienguyen) [inline README.md]: is this a convention for commits, pull requests, or both?

Comment (bphenriques) [inline README.md]: I believe both

Comment (bphenriques) [inline README.md]: The PR description is mostly the sum of the commit descriptions (exceptions may apply)

**Pull request merged**